content/LICENSE.md

```diff
@@ -1,7 +1,11 @@
-Content from MDN, found in this directory, is by Mozilla Contributors and is
+Prose from MDN, found in this directory, is by Mozilla Contributors and is
```

Comment:
Maybe "...licensed under [CC-BY-SA 2.5](http://creativecommons.org/licenses/by-sa/2.5/) unless otherwise incidcated"? There might still be some MDN content around that is not under this license; the [About MDN page](https://developer.mozilla.org/en-US/docs/MDN/About#Using_MDN_Web_Docs_content) specifically says "Unless otherwise indicated, the content is available under the terms of the Creative Commons Attribution-ShareAlike license (CC-BY-SA), v2.5 or any later version."

That's probably enough for now, IMO. We can figure out exactly how to indicate it at a later date.

Reply:
Done. Thanks!

Reply:
And my apologies for infecting your prose with that typo (I see you copied it straight across!)

"incidcated" -> "indicated" ;-)

Reply:
Pushing a fix now. I would've totally believed that was a typo of my own making 😆